spec/chrome_integration/gtalk.spec.js

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
Very confused. I think this same file is checked in twice?
spec/integration/gtalk.spec.js
spec/chrome_integration/gtalk.spec.js